pkg/client/cli/telepresence_test.go

```diff
@@ -303,6 +307,11 @@ func (ts *telepresenceSuite) TestA_WithNoDaemonRunning() {
 		ctx = filelocation.WithAppUserLogDir(ctx, tmpDir)
 		_, stderr := telepresenceContext(ctx, "connect")
 		require.Empty(stderr)
+		if goRuntime.GOOS == "windows" {
+			// Windows needs some time to get its bearings
+			time.Sleep(10 * time.Second)
+		}
```

Comment:
This at least needs a "FIXME" or a "BUG" comment; `telepresence connect` should block until it is set up.